config/secrets.yml

```diff
@@ -13,6 +13,8 @@
 development:
   secret_key_base: 34cced669c08639dad1fe8e42ef5f8b2d0096b5e62a4520d513e6d745dd814f7c14e013105304208192ff160e5ec5b2437545a6cf081cd0127d64ba1c027af39
   GGMAP_APP_KEY: <%=ENV['GGMAP_APP_KEY'] %>
+  hello_var: 'hello'
```

Comment:
We probably don't need this anymore since we only used it to test getting secret variables.